docs/usermanual.md

```diff
@@ -116,10 +116,36 @@ tools available online like [this one](https://string-functions.com/string-hex.a
 
 It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with
 `payloadType` `ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure
-attributes and `id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used,
-as well as `actionType`.
+attributes. The entity `id` and `type` are also include as attributes `ngsi_id` and `ngsi_type` (both with attribute
+type `string`). Destination entity `id` and entity `type` from device/group configuration provisioned are used,
```

Comment:
The default type is going to change from "string" to "Text" (issue https://github.com/telefonicaid/iotagent-json/issues/810). It seems to be a change to be done in the very short term, so maybe we can wait for it and align this PR documentation.

Reply:
@Keshav-NEC when do you plan to fix #810 please? I understand it should be a quick fix.

Reply:
Hi @fgalan, I am working on fix. While changing **`constants.DEFAULT_ATTRIBUTE_TYPE`** from `string `to `Text`, some test case were failing because of Assertion Error. 

> I will raise a PR with fix by February 2,2024. 

To fix `FUNCTIONAL TESTS` we have to do changes in [iotagent-node-lib](https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/functional/testCases.js) because the path for `FUNCTIONAL TESTS` is mentioned in [iotagent-json](https://github.com/Keshav-NEC/iotagent-json/blob/87686a37cc31e434579ca1efca7e2f5bfe9927d8/test/functional/functional-tests-runner.js#L44).

> Can we create an issue for that on iotagent-nodelib?

Reply:
It should be an straighforward fix in the test especations. Just change the place in which the usage of `"string"` is causing assertion fails to `"Text"`.

Just include tha fix also in your pull request and that should suffice.